content/webapp/pages/exhibition-guide.tsx

```diff
@@ -98,9 +115,14 @@ type StopsProps = {
 };
 
 const Stops: FC<StopsProps> = ({ stops, type }) => {
+  const stopWidth = type === 'bsl' ? 'calc(50% - 50px)' : 'calc(33% - 50px)';
```

Comment:
…oh I see, this is a bit more involved. Maybe just totally ignore my above comment.